index.html

```diff
@@ -19,12 +19,15 @@
                     </a>
                 </div>  
                 <div class="header-menu">
-                    <nav>
+                    <div class="menu-icon">
+                        <div class="icon"></div>
+                    </div>
+                    <nav class="menu-body">
```

Comment:
Please move the nav (and button with icons list) out from the header as was discussed